analysis/src/HistoManager.cxx

```diff
@@ -95,6 +95,10 @@ void HistoManager::DefineHistos(){
 void HistoManager::DefineHistos(std::vector<std::string> histoCopyNames, std::string makeCopyJsonTag){
     if (debug_ > 0) std::cout << "[HistoManager] DefineHistos" << std::endl;
     std::string h_name = "";
+    for (auto hist : _h_configs.items()){
```

Comment:
please remove this loop, or at least require debug_ > 0 for it.

Reply:
Done